### PR TITLE
Get Started Draft 

### PR DESCRIPTION
**Pull request by troft**
Here is the first draft of the BXML overview/get started guide. I add…ed some changes that I don't know how to do in the trello comments. Please let me know what I am missing or should change.

### CODE REVIEW

**Comment by dtolb** [comment]
Preview Changes at:
        http://bw-apdocs-BXMLStartupGuide.s3-website-us-east-1.amazonaws.com/

**Comment by dtolb** [comment]
Preview Changes at:
        http://bw-apdocs-BXMLStartupGuide.s3-website-us-east-1.amazonaws.com/

**Comment by dtolb** [comment]
Preview Changes at:
        http://bw-apdocs-BXMLStartupGuide.s3-website-us-east-1.amazonaws.com/

**Comment by dtolb** [comment]
Preview Changes at:
        http://bw-apdocs-BXMLStartupGuide.s3-website-us-east-1.amazonaws.com/

**Comment by dtolb** [comment]
Preview Changes at:
        http://bw-apdocs-BXMLStartupGuide.s3-website-us-east-1.amazonaws.com/

File: bxml/bxmlOverview.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+## Overview
+
+BXML is a much simpler way to interact with a call. Rather than working with callbacks, BXML handles the call state for you so all you have to implement is the order of events. BXML has many built in features and is perfect for quick development.
+
+## Assumptions
+
+* You have a way to recieve way to recieve webhooks.
+* You have downloaded [Postman](https://www.getpostman.com) or have another way to make Post requests to the API.
+* You have your [Voice and Messaging API](https://app.bandwidth.com) {userId}, {apiToken}, {apiSecret}
```
Comment (dtolb) [inline bxml/bxmlOverview.md]: Link to how to get userid, apiToken, apiSecret

File: bxml/bxmlOverview.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+	"callbackHttpMethod" : "GET"
+}
+```
+
+## Handle Incoming Calls
+**Make sure auto answer is on**
+### Using the Voice and Messaging Dashboard
+
+To allow the program to answer incoming calls, login to the [app.bandwidth.com](https://app.bandwidth.com) dashboard. Either create a new [application](https://app.bandwidth.com/applications/manage/) or edit an existing [application](https://app.bandwidth.com/applications/manage/). The callback request method should be <code class="get">GET</code> and the application type should be set to Voice. Add a callback URL and toggle “Automatically answer incoming calls” to on. Finally, add a Bandwidth phone number to this application. This number will be the number users can call. When the user calls the bandwidth number, the number will answer and notify the callback url that there is an active call. From there, the program acts the same way as create call.
+![BXML Application Setup](bxmlApplicationSetup.png)
```
Comment (dtolb) [inline bxml/bxmlOverview.md]: Picture not showing up

image

Comment (troft) [reply bxml/bxmlOverview.md]: How should I fix it? It is visible when I click the preview changes link

Comment (dtolb) [reply bxml/bxmlOverview.md]: Doesn't look like it picture was added to git. Try checking `git status` on your local machine to to see if it's there just not added to the repo.  Also try adding a new line between the end of the paragraph and the `BXML Application Setup`

**Comment by dtolb** [comment]
Preview Changes at:
        http://bw-apdocs-BXMLStartupGuide.s3-website-us-east-1.amazonaws.com/

**Comment by dtolb** [comment]
Preview Changes at:
        http://bw-apdocs-BXMLStartupGuide.s3-website-us-east-1.amazonaws.com/

**Comment by dtolb** [comment]
Preview Changes at:
        http://bw-apdocs-BXMLStartupGuide.s3-website-us-east-1.amazonaws.com/

**Comment by dtolb** [comment]
Preview Changes at:
        http://bw-apdocs-BXMLStartupGuide.s3-website-us-east-1.amazonaws.com/

**Comment by bwdan** [comment]
[Create Developer Guide](https://trello.com/c/o9n0S3PV/113-create-developer-guide)

**Pull request merged**